tests/system/action/projector/test_project.py

```diff
@@ -52,132 +53,166 @@ def test_project(self) -> None:
             {
                 "ids": [23],
                 "content_object_id": "assignment/453",
-                "options": {},
+                "meeting_id": 1,
```

Comment:
I see: You've added a meeting id. Ok

Reply:
Yes, this was needed to be able to unproject normal users - if no projector id is given, there is no way to determine the meeting which wanted to unproject them.